src/Browscap/Data/Factory/UserAgentFactory.php

```diff
@@ -10,16 +10,11 @@ class UserAgentFactory
      * validates the $userAgentsData array and creates at least one Useragent object from it
      *
      * @param array[] $userAgentsData
-     * @param array   $versions
      * @param bool    $isCore
      *
-     * @throws \RuntimeException If the file does not exist or has invalid JSON
-     * @throws \LogicException   If required attibutes are missing in the division
-     * @throws \LogicException
-     *
      * @return UserAgent[]
      */
-    public function build(array $userAgentsData, array $versions, bool $isCore) : array
+    public function build(array $userAgentsData, bool $isCore) : array
```

Comment:
Since the signature is changing here, did you search for all cases where this would have been called?

Also, should be documented in the Readme?

Reply:
added to Readme